### ISSUE TITLE
Error: Cannot find module '../client/ansi/filters/ansi.js'

### ISSUE THREAD

**Issue by msemenistyi** [Strider-CD/strider#991]
Hi there! I'm getting a following error after updating to version 1.9.1:
```
Error: Cannot find module '../client/ansi/filters/ansi.js'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/root/.nvm/versions/node/v5.8.0/lib/node_modules/strider/lib/ansi.js:3:18)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/root/.nvm/versions/node/v5.8.0/lib/node_modules/strider/lib/routes/jobs/index.js:10:14)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/root/.nvm/versions/node/v5.8.0/lib/node_modules/strider/lib/app.js:42:19)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/root/.nvm/versions/node/v5.8.0/lib/node_modules/strider/main.js:9:11)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
```

**Comment by oliversalzburg** [comment]
Where do you get this error?

**Comment by msemenistyi** [comment]
I get it when I start strider right after startup

**Comment by knownasilya** [comment]
Fix should be in Strider 1.9.2
